pkg/webhook/validation/validate_service.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2020 The Knative Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"knative.dev/pkg/logging"
+)
+
+// ExtraServiceValidation runs extra validation on Service resources
+func ExtraServiceValidation(ctx context.Context, uns *unstructured.Unstructured) error {
```

Comment:
Maybe to be more explicit `RevisionTemplateValidation`?

Reply:
nit: there's a studder with this type and the package name

ie. `validation.RevisionTemplateValidation`

If we drop `validation` from the path (by moving the files) it could be something like

`webhook.ValidateRevisionTemplate` ?